Packages/Core/Editor/Drawers/AtomBaseReferenceDrawer.cs

```diff
@@ -79,53 +71,43 @@ public override float GetPropertyHeight(SerializedProperty property, GUIContent
 
             var innerProperty = property.FindPropertyRelative(usageData.PropertyName);
 
-            bool forceSingleLine = false;
```

Comment:
why was forceSingleLine deleted?
this is what @AdamRamberg  added as a fix in #447 I gues it got lost, due to the way this PR was created by overriding the changes from the previous PRs